ntpcheck/checker/chrony.go

```diff
@@ -181,6 +177,19 @@ func (n *ChronyCheck) ServerStats() (*ServerStats, error) {
 	return serverStats, nil
 }
 
+// Unix returns true if connected via a unix socket
+func (n *ChronyCheck) Unix() bool {
```

Comment:
Why bother with this heuristics, if we already have `chronyConn.local`?

Reply:
Good suggestion. I changed it.